### ISSUE TITLE
not open source

### ISSUE THREAD

**Issue by christian-korneck** [christianselig/Amplosion#15]
Thanks for this useful app. This is just a friendly note that calling something “fully open source” that actually isn’t under an open source license (in the sense of an OSI approved license) is most likely misleading for most people. A term like “shared source” might be more appropriate.

**Comment by aschrijver** [comment]
I agree. It is a great app. If you consider open sourcing, I'll add it to awesome-humane-tech curated list.

https://github.com/humanetech-community/awesome-humane-tech

**Comment by vfonic** [comment]
It's a great app done in an entertaining way. Great job!

To some people the meaning of the words "open source" means a lot. This is incorrect interpretation of those words. Yes, for average Joe, open source might mean "open to read the source", but in professional, software development world, "open source" means more than that, as stated above.
